<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bharathi" failures="1" tests="2" name="orangehr" time="138.232" errors="0" timestamp="2023-07-03T11:04:14 IST">
  <testcase classname="com.TestNGProject.Runner_page" name="page_Launch" time="138.158">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at com.TestNGProject.Runner_page.page_Launch(Runner_page.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- page_Launch -->
  <testcase classname="com.TestNGProject.Runner_page" name="page_Login" time="0.0">
    <skipped/>
  </testcase> <!-- page_Login -->
</testsuite> <!-- orangehr -->
